app/models/article.rb

```diff
@@ -1,3 +1,4 @@
 class Article < ApplicationRecord
   validates :title, :content, presence: true
```

Comment:
You are validating the presence of the content, However, I can't see any sad path or test for these

Reply:
We have just created and tested it :)